modules/Story/HeaderRenderer/Subtitle.module.scss

```diff
@@ -0,0 +1,19 @@
+.subtitle {
```

Comment:
I wonder why you didn't style the subtitle with global classes as you did with the title?